test/powershell/Modules/PSReadLine/PSReadLine.Tests.ps1

```diff
@@ -42,6 +42,13 @@ Describe "PSReadLine" -tags "CI" {
         (Get-PSReadLineKeyHandler | where { $_.Key -ceq "Alt+B" }).Function | Should Be SelectBackwardWord
     }
 
+    It "Should ignore case when using Function binding" {
+        Set-PSReadlineOption -EditMode Windows
+        $lowerCaseFunctionName = "yank";
+        Set-PSReadlineKeyHandler "Ctrl+k" -Function $lowerCaseFunctionName
```

Comment:
Can you use a different key that is unbound, preferably one that most keyboards don't even have? That way if someone ran this test from their interactive sessions, it won't mess up the editing experience.